modules/cooldowns.lua

```diff
@@ -794,16 +799,15 @@ function Cooldowns:UpdateGroupIcons(unit, group)
 		-- save detached group spells
 		local index = GladiusEx:GetUnitIndex(unit)
 		header_gs.unit_spells = header_gs.unit_spells or { ["unit"] = unit }
-		header_gs.unit_spells[index] = sorted_spells
 
 		-- make list of the spells of all the units
 		local detached_spells = {}
-		for i = 1, 5 do
-			local us = header_gs.unit_spells[i]
-			if us then
-				local dunit = us.unit

```

Comment:
This is bogus, and I think really this is the underlying bug. `us.bug` is never set because of a thinko a few lines earlier.
I have a pending fix locally where I try to change a bit less.